specification/purview/resource-manager/Microsoft.Purview/stable/2021-07-01/purview.json

```diff
@@ -0,0 +1,1958 @@
+      "properties": {
+        "cloudConnectors": {
+          "$ref": "#/definitions/CloudConnectors",
+          "description": "Cloud connectors.\r\nExternal cloud identifier used as part of scanning configuration."
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "Gets the time at which the entity was created.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdBy": {
+          "description": "Gets the creator of the entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdByObjectId": {
+          "description": "Gets the creators of the entity's object id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "endpoints": {
+          "description": "The URIs that are the public endpoints of the account.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/AccountEndpoints"
+            }
+          ],
+          "readOnly": true
+        },
+        "friendlyName": {
+          "description": "Gets or sets the friendly name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "managedResourceGroupName": {
+          "description": "Gets or sets the managed resource group name",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "managedResources": {
+          "description": "Gets the resource identifiers of the managed resources.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ManagedResources"
+            }
+          ],
+          "readOnly": true
+        },
+        "privateEndpointConnections": {
+          "description": "Gets the private endpoint connections information.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "Gets or sets the state of the provisioning.",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Moving",
+            "Deleting",
+            "SoftDeleting",
+            "SoftDeleted",
+            "Failed",
+            "Succeeded"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        },
+        "publicNetworkAccess": {
+          "description": "Gets or sets the public network access.",
+          "default": "Enabled",
+          "enum": [
+            "NotSpecified",
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "publicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "systemData": {
```

Comment:
This is not the right location for "systemData".  It should be at the top level of the resource as a sibling of "type", "name", and "id".  This should be fixed before promoting the API to GA

Reply:
we don't use OpenApi for swagger generation as of now (as we are on older version of .net core 2.1). We generate swagger using legacy and systemData is not present in the acceptable list of top level properties.

We will work on this migration post GA. Is it possible to get exception here?
cc: @annoakes

Reply:
The tool used for swagger generation does not matter here.  Please fix the swagger spec (manually if necessary) to represent the correct systemData contract.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
Well this is a little disapointing not to be caught earlier, we had this done, it passed validation, and got sign off by two arm reviewers.

[https://github.com/Azure/azure-rest-api-specs/pull/14411](https://github.com/Azure/azure-rest-api-specs/pull/14411)

Reply:
@shaileshsk94 lets create a new model for the new API version as it seems there are a few things that were missed on previous reviews

Reply:
There was no ARM API review performed on that PR.  The LintDiff error related to systemData on accounts was not resolved in that PR.  https://github.com/Azure/azure-rest-api-specs/pull/14411#issuecomment-840904074

Reply:
Done.